### ISSUE TITLE
Distance based roughness

### ISSUE THREAD

**Issue by Rbn3D** [kode80/kode80SSR#12]
Having blur/roughness calculated by per pixel roughness but also affected by ray distance (like UE4 does) would be amazing. Doesn't need to be fully physically correct I guess.

**Comment by RosaryMala** [comment]
Yeah, this is definately needed. Normally, with glossy reflections, things that are closer are clearer, and things further away are more blurred out.
